tkweb/apps/gallery/templates/album.html

```diff
@@ -43,19 +43,22 @@
   {% for file in files %}
   <div class="col-xs-6 col-sm-4 col-md-3">
     <a class="thumbnail" href="{% url 'image' gfyear=album.gfyear album_slug=album.slug image_slug=file.slug %}" alt="Se billede">
-      {% if file %}
-        {% if file.file.crop.253x253.url %}
-          <img src="{{ file.file.crop.253x253.url }}" alt="{{ album.title }}" />
-        {% else %}
-          <div class="placeholder253x253">
-            <p>{% bootstrap_icon "file" %}</p>
-          </div>
-        {% endif %}
-      {% else %}
-        <p>Albummet '{{ album.title }}' har ingen billeder endnu.</p>
-      {% endif %}
+      <div class="thumbcap"
+           {% if file.type == "I" %}
+           style="background-image: url('{{ file.file.crop.253x253.url }}')"
```

Comment:
Hvad er grunden til at bruge `background-image` frem for `<img>`? Jeg kan godt lide `padding-bottom:100%` tricket, men jeg ville foretrække `<img style=margin-bottom:-100%>` frem for `background-image` (fordi jeg generelt foretrækker `<img>` frem for `background-image`).

Reply:
Jeg er enig. Jeg kunne ikke få det til at virke med `img`. Jeg prøvede igen og fik det til at virke. Se 977620cf99a2032188e2c088c400cb7a2cb93d4d